### DISCUSSION TITLE
un-sandbox'ing keka web version on Mac OS X 10.13

### DISCUSSION THREAD

**Discussion by spiderman666** [aonez/Keka#1446] [Q&A] [answered]
Hi all,

I am getting a variation of the error reported at issue #1389 

When trying to [un-sandbox](https://github.com/aonez/Keka/wiki/Rar-compression) to enable RAR on web v1.3.8(5320) I get the following...

```
/Applications/Keka.app: replacing existing signature
/Library/Developer/CommandLineTools/usr/bin/codesign_allocate: for architecture arm64 object: /Applications/Keka.app/Contents/MacOS/kekabrotli malformed object (unknown load command 5)
/Applications/Keka.app: the codesign_allocate helper tool cannot be found or used
In subcomponent: /Applications/Keka.app/Contents/MacOS/kekabrotli

```
Is this a problem with the app subcomponent or my old system?

At the moment I am just bypassing with [this](https://github.com/aonez/Keka/issues/123#issuecomment-466965500).

Thanks!

**Comment by aonez** [comment]
Let me check that one. Are you running on Intel?

**Reply by spiderman666** [reply]
Hi @aonez , Yes .. Intel here.

**Reply by aonez** [reply]
Sorry I should have also asked what macOS (or Mac OS X) version.

**Reply by spiderman666** [reply]
OSX High Sierra 10.13.6

**Reply by aonez** [reply]
I'm afraid the app should be flattened and all amd64 code stripped for the codesign process to work there.

- If you have another more modern OS you can do the codesign there and copy/paste the app.
- Otherwise I'll try to do a simple code to strip the arm64 code, let me get back to you in a few minutes.

**Reply by spiderman666** [reply, marked as answer]
I figured it would most likely be due to the age of my OS and hardware.
I am ok with the `codesign -f -s - "/Applications/Keka.app"` work around as it gets it functioning for what I needed to do.

**Reply by aonez** [reply]
I've tried flattening all arm64 code but it still fails. So probably un-sandboxing as you already did is the best way.

**Reply by spiderman666** [reply]
No problem, thank you for your time and effort, greatly appreciated!
